sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.7 (Unreleased)
 
+### Breaking Changes
+
+- Removed `GetDfsUri` in all clients since they are currently implementation details.
```

Comment:
nit: use `()` to indicate it is a method rather than a type or enum/member value.
```suggestion
- Removed `GetDfsUri()` in all clients since they are currently implementation details.
```